common/prque/prque_byteSlice.go

```diff
@@ -0,0 +1,65 @@
+// CookieJar - A contestant's algorithm toolbox
+// Copyright (c) 2013 Peter Szilagyi. All rights reserved.
+//
+// CookieJar is dual licensed: use of this source code is governed by a BSD
+// license that can be found in the LICENSE file. Alternatively, the CookieJar
+// toolbox may be used in accordance with the terms and conditions contained
+// in a signed written agreement between you and the author(s).
+
+// This is a duplicated and slightly modified version of "gopkg.in/karalabe/cookiejar.v2/collections/prque".
+
+// Package prque implements a priority queue data structure supporting arbitrary
+// value types and int64 priorities.
+//
+// If you would like to use a max-priority queue, pass false for isMinPrque.
+// If you would like to use a min-priority queue, pass false for isMaxPrque.
+//
+// Internally the queue is based on the standard heap package working on a
+// sortable version of the block based stack.
+
+package prque
+
+import "container/heap"
+
+type PrqueByteSlice struct {
+	cont *sstackByte
```

Comment:
@winnie-byun 
I think you don't need to use `sstackByte `.
Please check below.
http://golang.site/go/article/208-Go%EC%97%90%EC%84%9C-Heap-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0